parcel/Example_AerosolActivation.jl

```diff
@@ -0,0 +1,74 @@
+import OrdinaryDiffEq as ODE
+import CairoMakie as MK
+import Thermodynamics as TD
+import CloudMicrophysics as CM
+import ClimaParams as CP
+
+# definition of the ODE problem for parcel model
+include(joinpath(pkgdir(CM), "parcel", "Parcel.jl"))
+FT = Float32
+
+# Get free parameters
+tps = TD.Parameters.ThermodynamicsParameters(FT)
+wps = CMP.WaterProperties(FT)
+
+# Initial conditions
+Nₐ = FT(1 * 1e6)
+Nₗ = FT(0)
+Nᵢ = FT(0)
+T₀ = FT(230)
+cᵥ₀ = FT(5 * 1e-5)
+ln_INPC = FT(0)
+
+# Constants
+ρₗ = wps.ρw
+R_v = TD.Parameters.R_v(tps)
+R_d = TD.Parameters.R_d(tps)
+ϵₘ = R_d / R_v
+eₛ = TD.saturation_vapor_pressure(tps, T₀, TD.Liquid())
+qᵥ = ϵₘ / (ϵₘ - 1 + 1 / cᵥ₀)
+# Compute qₗ assuming that initially droplets are lognormally distributed
+# with N(r₀, σ). We are not keeping that size distribution assumption
+# in the simulation
+r₀ = FT(3e-7)
+σ = FT(2)
+qₗ = Nₗ * FT(4 / 3 * π) * exp((6 * log(r₀) + 9 * σ^2) / (2))
+qᵢ = FT(0)
+Sₗ = FT(1.2)
+e = Sₗ * eₛ
+p₀ = e / cᵥ₀
+IC = [Sₗ, p₀, T₀, qᵥ, qₗ, qᵢ, Nₐ, Nₗ, Nᵢ, ln_INPC]
+
+# Simulation parameters passed into ODE solver
+w = FT(1)           # updraft speed
+const_dt = FT(1)    # model timestep
+t_max = FT(30)      # total time
```

Comment:
Would it be possible to start below supersaturation and run the simulation long enough to see the supersaturation reach its maximum value and then start decreasing (the same way as you would expect in the cloud going from cloud base aerosol activation inside the cloud conditions)